Gemfile

```diff
@@ -65,6 +65,7 @@ group :development, :test do
   gem 'dotenv-rails', '~> 2.4'
   gem 'site_prism', '~> 3.0'
   gem 'rspec_api_documentation', '~> 6.1'
+  gem 'bullet', '~> 5.9'
```

Comment:
Bullet was added mid flight to help me find performance issues as the test suite was slowing right down.